packages/slice-machine/components/ReviewModal/index.tsx

```diff
@@ -40,10 +40,11 @@ const ratingSelectable = [1, 2, 3, 4, 5, 6, 7];
 const SelectReviewComponent = ({ field, form }: FieldProps) => {
   return (
     <Box sx={{ mb: 3 }}>
-      {ratingSelectable.map((rating) => (
+      {ratingSelectable.map((index, rating) => (
```

Comment:
pretty sure index is the second parameter for map functions :)

Reply:
ohhhh you right :p

Reply:
you'll get the waring about using the index as the key, so it maybe better to use the rating for the key (assuming the key is unique)